### DISCUSSION TITLE
绳友们，最近沉迷甜旅了，有什么相似风格的专来推一推？

### DISCUSSION THREAD

**Discussion by maijiaosesu** [share121/inter-knot#474] [其他]
![微信图片_20240531224424](https://github.com/user-attachments/assets/335f6378-14f7-4c5d-947a-005f43bd4912)
我已经沉迷vdc无法自拔了♿♿♿一打开wyy全都是 **choose life？！**
This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

This time maybe it's true~~u~~u~~

**Comment by warning-Kirov-reporting** [comment]
这个话题可能有点小众

**Comment by maijiaosesu** [comment]
本来听sweet trip 已经很自卑了，可是我的自卑………………
